packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -189,9 +190,12 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
         <String>[sdkManagerPath, '--licenses'],
         environment: sdkManagerEnv,
     );
-    stdout.addStream(process.stdout);
-    stderr.addStream(process.stderr);
-    process.stdin.addStream(stdin);
+
+    waitGroup(<Future<Null>>[
+      stdout.addStream(process.stdout),
+      stderr.addStream(process.stderr),
+      process.stdin.addStream(stdin),
```

Comment:
Nice - this fixes a bug where we'd exit `flutter doctor --android-licenses` before the user had a chance to answer the pending question from the SDK manager 👍